src/pages/elements/2x-grid/overview.mdx

```diff
@@ -400,9 +400,9 @@ it to the top or bottom margin of a box:
 
   </DoDont>
 
-  <DoDont type="dont" caption="In Figma, always use “Auto width” in the Text panel to ensure the text box is fitting with the content. Spacer snaps to the text box.">
+  <DoDont type="dont" caption="In Figma, always use “Auto Height” in the Text panel to ensure the text box fits with the content. Spacer snaps to the text box.">
```

Comment:
```suggestion
  <DoDont type="dont" caption="In Figma, always use “Auto height” in the Text panel to ensure the text box fits with the content. Spacer snaps to the text box.">
```
Nice catch @chadcwimberly. Just suggesting this be in sentence case instead of title case like it is in Figma.

Reply:
good idea 👍🏻